prow/cmd/hook/Dockerfile

```diff
@@ -15,7 +15,7 @@
 FROM alpine:3.4
 MAINTAINER spxtr@google.com
 
-RUN apk add --no-cache ca-certificates && update-ca-certificates
+RUN apk add --no-cache git ca-certificates && update-ca-certificates
```

Comment:
git-sync is probably not working with PRs .... Also it would most likely be racing with the plugins..